swap/src/bin/asb.rs

```diff
@@ -78,6 +78,23 @@ async fn main() -> Result<()> {
             let bitcoin_wallet = init_bitcoin_wallet(&config, &seed, env_config).await?;
             let monero_wallet = init_monero_wallet(&config, env_config).await?;
 
+            let bitcoin_balance = bitcoin_wallet.balance().await?;
+            info!(
+                "Connection to Bitcoin wallet succeeded, balance: {}",
```

Comment:
I don't think this "Connection to Bitcoin wallet succeeded" part is very useful. Can we remove it? That would also nicely align the message with the Monero balance (where we indeed do connect to an external wallet, lol)